http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/FollowupComponent.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package db
+
+import org.broadinstitute.dsde.workbench.leonardo.monitor.ClusterFollowupDetails
+
+import scala.concurrent.ExecutionContext
+import LeoProfile.api._
+
+case class FollowupRecord(clusterId: Long, status: String, masterMachineType: Option[String])
+
+class FollowupTable(tag: Tag) extends Table[FollowupRecord](tag, "CLUSTER_FOLLOWUP") {
+  def clusterId = column[Long]("clusterId", O.PrimaryKey)
```

Comment:
which column this maps to in `CLUSTER`?

Reply:
this maps directly to the id field in the cluster table via the foreign key constraint on line 16